style.css

```diff
@@ -0,0 +1,27 @@
+*{
+    margin: 0;
+    padding: 0;
+}
+
+
+table {
+    font-family: arial, sans-serif;
+    border-collapse: collapse;
+    width: 65%;
+  }
+  
+  td, th {
+    border: 1px solid #928291;
+    text-align: left;
+    padding: 8px;
+  }
+  
+  tr:nth-child(even) {
```

Comment:
The background color for the even tds are overlapping with the font color 
![image](https://user-images.githubusercontent.com/57461706/188811039-4a3f74d3-0259-406a-922a-f241acda5f3e.png)